odl/operator/pspace_ops.py

```diff
@@ -776,7 +852,7 @@ def prod_op(self):
 
     @property
     def operators(self):
-        """A tuple of sub-operators."""
+        """`tuple` of sub-operators."""
```

Comment:
Well, writing `tuple` implies that it actually is of type `tuple` and not simply a tuple of operators. This is again a case of english vs python. I guess this could be made clear by adding a `Returns`, but this is likely enough.